pkgs/by-name/mo/monaspace/package.nix

```
@@ -58,4 +33,35 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     maintainers = [ ];
     platforms = lib.platforms.all;
   };
+  makeFont =
+    font:
+    stdenvNoCC.mkDerivation {
+      inherit (font) pname;
+      inherit (fonts) version;
+      src = fetchzip {
+        url = "${baseUrl}/v${fonts.version}/${font.pname}-v${fonts.version}.zip";
+        stripRoot = false;
+        inherit (font) hash;
+      };
+
+      installPhase = ''
+        runHook preInstall
+
+        # The result of unzipping the file is a folder containing a folder containing the font file/-s
+        install -Dm644 **/**/*.* -t $out/share/fonts/${font.destination}
```

The comment I made about removing `stripRoot = false` affects this.

Also, instead of installing everything at a certain level of the hierarchy, assuming that it is all fonts, only install those files with the file extension that we want.

```suggestion
        find . -type f \( -name "*.otf" -o -name "*.ttf" -o -name "*.woff" \) \
          -exec install -Dm644 {} $out/share/fonts/${destination} \;
```